### PR TITLE
Write data generation script

### PR DESCRIPTION
**Pull request by swiimp**

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -1,70 +1,71 @@
...
 const app = express();
 const bodyParser = require('body-parser');
 const path = require('path');
 const db = require('../db/mongodb.js');
 const mongoose = require('mongoose');
 
-var mongoUrlDocker = 'mongodb://database/apateez-nearby';
-var mongoUrl = 'mongodb://localhost/apateez-nearby';
+const mongoUrlDocker = 'mongodb://database/apateez-nearby';
+const mongoUrl = 'mongodb://localhost/apateez-nearby';
```
Comment (zacharyhill) [inline server/index.js]: Maybe consider making these environment variables

File: server/index.js [hunk 2]
```diff
@@ -1,70 +1,71 @@
...
+mongoose.connection.on('connected', () => {
+  console.log('Mongoose connection open');
 });
 
-mongoose.connection.on('error',function (err) {
-  console.log('Mongoose default connection error: ' + err);
-  mongoose.connect(mongoUrlDocker)
+mongoose.connection.on('error', (err) => {
+  console.log(`Mongoose default connection error: ${err}`);
+  mongoose.connect(mongoUrlDocker);
```
Comment (zacharyhill) [inline server/index.js]: I know this was inherited, but maybe we should remove the part that tries to connect to a different database on any error.

File: server/index.js [hunk 3]
```diff
@@ -1,70 +1,71 @@
...
 });
 
 app.use(bodyParser.json());
 
-app.use(function(req, res, next) {
- res.header("Access-Control-Allow-Origin", "*");
- res.header("Access-Control-Allow-Headers", "Origin, X-Requested-With, Content-Type, Accept");
- next();
+app.use((req, res, next) => {
```
Comment (zacharyhill) [inline server/index.js]: Consider cleaning this up by using the cors npm package

**Comment by swiimp** [comment]
I have a few more things to add before trying to merge this

**Pull request closed without merging**